### DISCUSSION TITLE
Micro invert

### DISCUSSION THREAD

**Discussion by bsnel** [kellerza/sunsynk#293] [General]
My solar energy is coming from my Solaredge inverter to the Deye inverter through the micro inverter port. If someone knows if this is a smart setup or not, keen to hear.

my question is, can i get the reading from the microinverter through this HA plugin? Screenshot indicates what it looks like in my Solarman dashboard

![IMG_3185](https://github.com/kellerza/sunsynk/assets/31987657/b84b8e07-397e-440f-b4f2-73455d895f67)

**Comment by concierge-at** [comment]
Yes, you can, unfortunately it is shown negatively so that the flow direction is correct, I'm sure there is also a solution here

**Reply by bsnel** [reply]
What do you mean “yes, you can”? Please tell me how as I don’t see the sensor anywhere. No problem to invert the values to show correct flow

**Reply by andyholmes999** [reply]
Did you find a solution to this please?  I dont see the entity being pulled from Sunsynk either.

**Reply by concierge-at** [reply]
I'm sorry, I no longer use this integration, so I can't say anything more about it

**Comment by kellerza** [comment]
Is this a micro inverter port, or the AUX port being used as another input?

**Reply by unwiredtech** [reply]
Hi,

we have a gen port that can be used as either for generator or micro inverter and a smart load. The add on can detect this? If for example every day I use it for micro inverter then there is a strong typhoon and the grid went down so I will augment my deye with a generator (micro inverter and generator via Transfer switch).
